src/Graphics/Canvas.purs

```diff
@@ -121,10 +118,7 @@ foreign import data CanvasElement :: *
 foreign import data Context2D :: *
 
 -- | An image data object, used to store raster data outside the canvas.
-foreign import data ImageData :: *
-
--- | An array of pixel data.
-foreign import data CanvasPixelArray :: *
+type ImageData = { width :: Number, height :: Number, data :: Uint8ClampedArray }
```

Comment:
The docs say `height` is "an unsigned long representing the actual height" for example. I think we can use `Int` here.

The reason for the current code is that `Number` was the only option back when this was added.